custodial-copy-backend/src/test/scala/uk/gov/nationalarchives/custodialcopy/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -595,10 +632,17 @@ object ExternalServicesTestUtils extends MockitoSugar with EitherValues {
     when(sqsClient.deleteMessage(ArgumentMatchers.eq("queueUrl"), ArgumentMatchers.startsWith("receiptHandle")))
       .thenReturn(IO.pure(DeleteMessageResponse.builder.build))
 
-    val ocflService: OcflService = mockOcflService(missingObjects, changedObjects, pathsOfObjectsUnderIo, throwErrorInMissingAndChangedObjects)
+    val ocflService: OcflService = mockOcflService(entityType, objectHasChanged, pathsOfObjectsUnderIo, throwErrorInMissingAndChangedObjects)
     val xmlValidator: ValidateXmlAgainstXsd[IO] = spy(ValidateXmlAgainstXsd[IO](XipXsdSchemaV7))
 
     val processor: Processor = new Processor(config, sqsClient, ocflService, entityClient, xmlValidator, snsClient)
+
+    val processMessage: IO[Result] = entityType match {
+      case ContentObject     => processor.process(coMessageResponse)
+      case InformationObject => processor.process(ioMessageResponse)
+      case unsupportedObject => throw new Exception(s"Unsupported object: $unsupportedObject")
```

Comment:
should there be a test for the default behaviour?

Reply:
This is the test utils; this case should never be reached since we're not testing for anything other than COs and IOs,  and if we do decide to, then this exception will be thrown and someone will implement it. The reason why I added this case is because IntelliJ kept warning me about an non-exhaustive match